allel/util.py

```diff
@@ -145,6 +146,36 @@ def ensure_square(dist):
     return dist
 
 
+def mask_inaccessible(is_accessible, ac, pos):
+    """ This function removes any positions that are not accessible from the
+    allele counts Array and the vector of variant positions.
+
+    Parameters
+    ----------
+    is_accessible : array_like, bool, shape (len(contig),), optional
```

Comment:
not optional

Reply:
emphasize that this should have a boolean for every position (not jsut variants)

Reply:
I think it is well described the way it is "Boolean array indicating accessibility status for all positions in the chromosome/contig."